src/urlschemehandler.cpp

```diff
@@ -49,7 +49,8 @@ UrlSchemeHandler::handleContentRequest(QWebEngineUrlRequestJob *request)
     BlobBuffer* buffer = new BlobBuffer(entry.getBlob());
     auto mimeType = QByteArray::fromStdString(entry.getMimetype());
     mimeType = mimeType.split(';')[0];
-    connect(buffer, &QIODevice::aboutToClose, buffer, &QObject::deleteLater);
+    // connect(buffer, &QIODevice::aboutToClose, buffer, &QObject::deleteLater);
+    connect(request, &QObject::destroyed, buffer, &QObject::deleteLater);
```

Comment:
it seems that the deletion of the buffer is the cause of the random crash
According to the doc, it can be deleted like this.

![image](https://user-images.githubusercontent.com/47149116/59093428-3733be80-8914-11e9-83a4-5d11e0c60b00.png)

https://doc.qt.io/qt-5/qwebengineurlrequestjob.html#reply